### PR TITLE
Add tests to increase code coverage

### PR DESCRIPTION
**Pull request by yanvow**
Add tests for the login UI to increase code coverage

### CODE REVIEW

File: app/src/main/java/com/github/factotum_sdp/factotum/MainActivity.kt [hunk 1]
```diff
@@ -34,7 +36,9 @@ class MainActivity : AppCompatActivity() {
 
         val drawerLayout: DrawerLayout = binding.drawerLayout
         val navView: NavigationView = binding.navView
-        val navController = findNavController(R.id.nav_host_fragment_content_main)
```
Comment (schuetzcarl) [inline app/src/main/java/com/github/factotum_sdp/factotum/MainActivity.kt:37]: I don't get why it's a refactor here ? It seems that these two new lines does the job of one.

File: app/src/main/res/layout/content_main.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
     app:layout_behavior="@string/appbar_scrolling_view_behavior"
     >
 
-    <fragment
+    <androidx.fragment.app.FragmentContainerView
```
Comment (schuetzcarl) [inline app/src/main/res/layout/content_main.xml:9]: Could you motivate that change ?

**Review by schuetzcarl — CHANGES_REQUESTED** [review]
Thank you for fixing these tests and the for the additional clean up you have made.

I only have two small remarks, then I will agree to merge it.

File: app/src/main/res/layout/content_main.xml (hunk 2 above)
Comment (yanvow) [reply app/src/main/res/layout/content_main.xml:9]: - It is google’s recommended view for hosting fragments.
- Fragment transactions are getting handled by FragmentContainerView under the hood.

File: app/src/main/java/com/github/factotum_sdp/factotum/MainActivity.kt (hunk 1 above)
Comment (yanvow) [reply app/src/main/java/com/github/factotum_sdp/factotum/MainActivity.kt:37]: It's the way to have a navController when you have a FragmentContainerView

File: app/src/main/res/layout/content_main.xml (hunk 2 above)
Comment (schuetzcarl) [reply app/src/main/res/layout/content_main.xml:9]: Ok let's try. It seems to have also some nice animation methods for fragments transition.

**Review by schuetzcarl — APPROVED** [review]

**Pull request merged**